account_cutoff_base/models/account_cutoff.py

```diff
@@ -0,0 +1,358 @@
+# © 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class AccountCutoff(models.Model):
+    _name = 'account.cutoff'
+    _rec_name = 'cutoff_date'
+    _order = 'cutoff_date desc'
+    _inherit = ['mail.thread']
+    _description = 'Account Cut-off'
+
+    @api.multi
+    @api.depends('line_ids', 'line_ids.cutoff_amount')
+    def _compute_total_cutoff(self):
+        for cutoff in self:
+            tamount = 0.0
+            for line in cutoff.line_ids:
+                tamount += line.cutoff_amount
+            cutoff.total_cutoff_amount = tamount
+
+    @api.model
+    def _default_move_label(self):
+        type = self._context.get('type')
```

Comment:
`s/type/type_` to avoid clash w/ built-in `type`.
`s/self._context/self.env.context`